_layouts/standalone_resource.html

```diff
@@ -200,7 +200,8 @@ <h1>
               Rule Mapping 
             </header>
             <div class="box-i"> 
-             <p><strong>{{ page.rule_meta.rule_type | capitalize }} rule</strong> 
+             <p>This is {% if page.rule_meta.rule_type == "atompic" %}an {% else %}a {% endif %}
```

Comment:
Atompic -> atomic ???
```suggestion
             <p>This is {% if page.rule_meta.rule_type == "atomic" %}an {% else %}a {% endif %}
```